server/room-battle.ts

```diff
@@ -568,6 +568,7 @@ export class RoomBattle extends RoomGames.RoomGame {
 		}
 		this.timer = new RoomBattleTimer(this);
 		if (Config.forcetimer) this.timer.start();
+		if (this.format.includes('blitz')) this.timer.start();
```

Comment:
Why isn't this an `else`?

Reply:
Heck, why isn't this an `||`?

Reply:
because i'm dense!

Reply:
that is... much better and i'll set it to that now, not sure why i didn't.. remember that exists